BaseClasses.py

```diff
@@ -1389,6 +1390,14 @@ def write_option(option_key: str, option_obj: Options.AssembleOptions) -> None:
 
             AutoWorld.call_all(self.multiworld, "write_spoiler", outfile)
 
+            precollected_items = [f"{item.name} ({self.multiworld.get_player_name(item.player)})"
+                                  if self.multiworld.players > 1
+                                  else item.name
+                                  for item in chain.from_iterable(self.multiworld.precollected_items.values())]
+            if precollected_items:
+                outfile.write("\n\nPrecollected Items:\n\n")
```

Comment:
I wonder whether this should just say "starting inventory," I'm not sure if there is value in distinguishing by this labeling between "the content of the start_inventory option" and "your actual starting inventory as it appears in precollected_items." But I'm not very strongly opinionated either way so I will leave this for others to give their thoughts